File: Function Key/function_key.py
import requests

url = "https://your-azure-function-app.azurewebsites.net/api/your-function-name"
function_key = "your_azure_function_key_here"

params = {"code": function_key}

response = requests.get(url, params=params)
print(response.text)

File: Function Key/function_key.js
const axios = require('axios');

const url = 'https://your-azure-function-app.azurewebsites.net/api/your-function-name';
const function_key = 'your_azure_function_key_here';

const params = {
    code: function_key
};

axios.get(url, { params })
    .then(response => {
        console.log(response.data);
    })
    .catch(error => {
        console.error(error);
    });